ygot/pathstrings.go

```diff
@@ -46,6 +46,33 @@ func PathToString(path *gnmipb.Path) (string, error) {
 	return "/" + stdpath.Join(s...), err
 }
 
+// PathToSchemaPath returns the supplied Path as its corresponding schema path.
+// The YANG schema path removes any keys (i.e., predicates) from the path, using
+// only the name.
+func PathToSchemaPath(path *gnmipb.Path) (string, error) {
```

Comment:
This is a great way of making this change as it is backwards compatible with all pre-existing code.

Thank you for looking into this.

Reply:
No worries! Thanks.